BrightcoveExoPlayer-Samples/README.md

```diff
@@ -0,0 +1,88 @@
+BrightcoveExoPlayer-Samples
+======================
+
+**_Note: The BrightcoveExoPlayer integration used in these sample applications is in beta._**
+
+## Description
+
+A collection of sample applications using the Brightcove Player SDK for Android with the BrightcoveExoPlayer integration.
+
+These sample applications are built to be modified and run from Android Studio and support the latest Brightcove SDK and plugins.
+The following sample applications are included:
+
+* BasicExoPlayerSampleApp: Creates an instance of the BrightcoveExoPlayer integration and demonstrates playback functionality for an MP4 video.
+
+## Features
+
+The BrightcoveExoPlayer integration currently supports the following:
+* HLS playback
+* EIA-608 embedded captions
+* WebVTT and TTML sidecar captions
+
+## Requirements
+
+The BrightcoveExoPlayer integration requires:
+* Android version 4.1 and above (API level 16 and up)
+
+The BrightcoveExoPlayer integration is currently developed with Android Studio and Gradle. Eclipse is not officially supported.
```

Comment:
It would be good to advertise which minimum version of AS is required.  Actually a range at the time of release might be better, i.e. tested with AS version from a.b to x.y.